presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/env/Environments.java

```diff
@@ -109,4 +158,45 @@ public static String nameForConfigClass(Class<? extends EnvironmentConfig> clazz
     {
         return CaseFormat.UPPER_CAMEL.to(CaseFormat.LOWER_HYPHEN, clazz.getSimpleName());
     }
+
+    public static String networkName(String environmentName)
+    {
+        return format("%s.%s", environmentName, PRODUCT_TEST_ENVIRONMENT);
+    }
+
+    public static String containerId(String environmentName, String containerName)
+    {
+        return format("%s-%s", environmentName, containerName);
```

Comment:
i often use the fact that the containers have fixed, easy name

see https://github.com/prestosql/presto/pull/4834#discussion_r476354288

Reply:
I've zsh function for that. Is it a big deal for you?

Reply:
I can reverse the name (containerName-environmentName) which will make it easier to autocomplete

Reply:
Would it help me to run `docker exec -it ptl-hadoop-master bash -l` easily?
often in a different shell than the one that was starting the environment?

Reply:
> I can reverse the name (containerName-environmentName) which will make it easier to autocomplete

interesting idea!

fwiw, i am heavy history user (ctrl+r in bash)

Reply:
Also, if i had 2 envs running side by side, i'd prefer env name to be prefix